<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>transformWindow</class>
 <widget class="QMainWindow" name="transformWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>644</width>
    <height>635</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>transformWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>621</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>源</string>
    </property>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>161</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>请选择Word文档所在目录：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="sourcepath">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>30</y>
       <width>381</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="sourcebrowseButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listword">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>320</y>
      <width>621</width>
      <height>261</height>
     </rect>
    </property>
    <property name="title">
     <string>结果</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>151</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>转换后PDF文件保存目录：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="targetpath">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>30</y>
       <width>391</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QToolButton" name="targetbrowseButton">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>30</y>
       <width>31</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QListWidget" name="listpdf">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>60</y>
       <width>581</width>
       <height>192</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="showLoading">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>289</y>
      <width>431</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>转换进度</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multipeExecute">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>280</y>
      <width>81</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>批量转换</string>
    </property>
   </widget>
   <widget class="QPushButton" name="singleExecute">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>280</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>合并为一个PDF</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>644</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
